### ISSUE TITLE
suggest K8S heml esinit job change CronJob

### ISSUE THREAD

**Issue by nulltrue** [apache/skywalking#5007]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [ ] Bug
- [ ] Requirement
- [x] Feature or performance improvement

___
### Question
- What do you want to know?

___
### Bug
- Which version of SkyWalking, OS and JRE?

- Which company or project?

- What happen?
If possible, provide a way for reproducing the error. e.g. demo application, component version.

___
### Requirement or improvement
- Please describe about your requirements or improvement suggestions.
When SkyWalking is deployed in K8S,esinit is in job form and is only executed once against Skywalking Heml, and if there are no more queries that day, the ALARM_Record in S will not be executed.The ALARM_Record will continue to be authenticated by the OAP causing the POD to fail to start and get stuck in: table: ALARM_Record does not exist.The OAP runs in "no-init" mode instead of the CronJo, which is waiting, and the ALARM_Record can be rebuilt periodically to handle this issue, and the CronJo can be executed manually at any time to see if the scheme is viable.
You can replace
https://github.com/apache/skywalking-kubernetes/blob/master/chart/skywalking/templates/es-init.job.yaml
content;
Attached is the YAML file of CronJo I am using:
```
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  labels:
    app: skywalking
    chart: skywalking-2.0.0
    component: mysw-v2-skywalking-job
    heritage: Helm
    release: mysw-v2
  name: mysw-v2-es-init-cj
  namespace: skywalking
spec:
  schedule: "0 1 * * *"
  jobTemplate:
    spec:
      template:
        metadata: {}
        spec:
          containers:
            - env:
                - name: JAVA_OPTS
                  value: -Xmx2g -Xms2g -Dmode=init
                - name: SW_STORAGE
                  value: elasticsearch7
                - name: SW_STORAGE_ES_CLUSTER_NODES
                  value: XXXXX:9200
              image: apache/skywalking-oap-server:7.0.0-es7
              imagePullPolicy: IfNotPresent
              name: skywalking-oap
              resources: {}
              terminationMessagePath: /dev/termination-log
              terminationMessagePolicy: File
          restartPolicy: Never
          schedulerName: default-scheduler
          securityContext: {}
          terminationGracePeriodSeconds: 30
```

**Comment by wu-sheng** [comment]
This is the TTL bug, which has been fixed in the 8.0 release.

**Comment by wu-sheng** [comment]
TTL timer wouldn't delete the last index, which causes the cases.
